src/network/raknet/handler/INetworkQueueHandler.ts

```diff
@@ -0,0 +1,3 @@
+export default interface INetworkQueueHandler {
```

Comment:
Should probably be named `NetworkQueueHandlerType` to follow the standard set in the rest of the codebase :)